app/Console/Commands/GenerateSitemap.php

```
@@ -124,6 +126,19 @@ public function handle(): int
                 );
             });
 
+        // 6. Published Forum Questions
+        ForumPost::where('is_published', true)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\ForumPost::where().


(staticMethod.notFound)